docs/docs/deployment/pre_deployment/prerequisites.md

```diff
@@ -1,14 +1,20 @@
 ---
-id: prerequisites
-title: Prerequisites
-sidebar_label: Prerequisites
+id: accounts
+title: Accounts
+sidebar_label: Accounts
 ---
 
 ## Deployment Account
-------------
-Service Workbench can be deployed to an existing AWS Organization master account, or a member account, or some other account not part of the AWS Organization.
+
+Service Workbench can be deployed from either a master or member account of an AWS Organization, or from an account not associated with an Organization.  The choice is determined by customer preference.  The account in which Service Workbench is deployed, is known as the Service Workbench *Main* account.  During the installation you will require access to this main account, either through direct login, or through an AWS Command Line Interface [Profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)
+
+## Billing Account
+
+Within Service Workbench, all resources are deployed against an account which is responsible for charges incurred by that resource.  Billing accounts can be either **Created** by Service Workbench (vending an account within the Organization), or can be **Added** (importing an existing external account for billing purposes).  There can be multiple billing accounts set up within Service Workbench, with projects created under each account.
```

Comment:
Is 'Billing Account' the same as 'Member Account'?

Reply:
I use Master/Member only in the context of an Organization.  I use the term 'Billing account' for an Account that has been added to SWB through the 'Add Account' mechanism.  The term 'Account' is used 5 different ways in SWB...so I used the term 'Billing account' here.  Perhaps saying 'These billing accounts can be...' makes it clearer that it refers to Added or Created accounts.